feature_server/commands.py

```diff
@@ -197,10 +213,16 @@ def say(connection, *arg):
     connection.protocol.send_chat(value)
     connection.protocol.irc_say(value)
 
-@admin
-def kill(connection, value):
+add_rights('kill', 'admin')
```

Comment:
Previous version: only admins can use
Proposed version: regular players can commit suicide and admin can kill whoever they want